test/framework/src/main/java/org/elasticsearch/test/AbstractBuilderTestCase.java

```diff
@@ -670,5 +673,9 @@ private ResolvedIndices createMockResolvedIndices() {
                 Map.of(index, indexMetadata)
             );
         }
+
+        private QueryRewriteInterceptor createMockQueryRewriteInterceptor() {
+            return new MockQueryRewriteInterceptor();
```

Comment:
Shouldn't `MockQueryRewriteInterceptor` class be part of test framework?

Reply:
It is?

Reply:
My bad - GH shortened the path and it seemed to be src/main 🤷